### ISSUE TITLE
AssertionError for `assert(Ex_err < 1e-14)`, when I run test_fft_poisson_3d.py

### ISSUE THREAD

**Issue by gitansh95** [QuazarTech/Poisson_Solver#2]
When I run `test_fft_poisson_3d.py`, I get the following error :
```
N_x 18
N_y 25
N_z 17
Ex_err :  0.001128741825092327
Ey_err :  0.0022574836501846555
Ez_err :  0.044020931178600606
Traceback (most recent call last):
  File "test_fft_poisson_3d.py", line 77, in <module>
    test_fft_poisson()
  File "test_fft_poisson_3d.py", line 73, in test_fft_poisson
    assert(Ex_err < 1e-14)
AssertionError
```
The `N_x`, `N_y` and `N_z` variables are being generated randomly inside the code using 
```
N_x = np.random.randint(16, 32)
N_y = np.random.randint(16, 32)
N_z = np.random.randint(16, 32)
```
So the error is not reproduced each time the code is run. I printed out the values of `N_x`, `N_y`, `N_z`, `Ex_err`, `Ey_err` and  `Ez_err` as seen in the above error.

**Comment by gitansh95** [comment]
I would also like to report two more of these : 
```
N_x 30
N_y 28
N_z 17
Ex_err :  0.0011287415971804864
Ey_err :  0.0022574831943609733
Ez_err :  0.04402092229003841
Traceback (most recent call last):
  File "test_fft_poisson_3d.py", line 77, in <module>
    test_fft_poisson()
  File "test_fft_poisson_3d.py", line 73, in test_fft_poisson
    assert(Ex_err < 1e-14)
AssertionError
```
and

```
N_x 18
N_y 31
N_z 18
Ex_err :  3.685961650771807e-17
Ey_err :  5.40760618644171e-17
Ez_err :  0.03410463066255147
Traceback (most recent call last):
  File "test_fft_poisson_3d.py", line 77, in <module>
    test_fft_poisson()
  File "test_fft_poisson_3d.py", line 75, in test_fft_poisson
    assert(Ez_err < 1e-14)
AssertionError

```

**Comment by shyams2** [comment]
Fixed in #4
